frontend/scss/pages/_p-landingpage-show.scss

```diff
@@ -6,6 +6,24 @@
 
 .visit {
 
+  .m-title-bar {
+    margin-top: 50px;
+  }
+
+  .o-blocks .hr {
+    margin-top: 40px !important;
+  }
+
+  .select__select select {
+    font-family: "Ideal Sans A","Ideal Sans B",Helvetica,Arial,sans-serif;
```

Comment:
Use `$sans-serif-font--loaded` as a value here. So if we ever tweak fonts, we only have to do in in one place.